distribution/readme.txt

```diff
@@ -69,7 +69,7 @@ following information in your bug report:
 3.0) Supported platforms
 ---- -------------------
 OpenRCT2 is currently supported on Windows Vista and above, many distributions of
-Linux and OSX 10.7. OpenRCT2 will only work on x86 architectures as the
+Linux and macOS 10.7. OpenRCT2 will only work on x86 architectures as the
```

Comment:
Technically it's still OS X for older versions.  Only Sierra will be macOS, and I'm kind of wondering if they're even going to assign a number at all.